features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/api-subscriptions.jag

```diff
@@ -18,7 +18,7 @@
  *
  */
 require('/modules/store.js').exec(function (ctx) {
-    var log = new Log();
+    var log = new Log('api_subsciptions_endpoint');
```

Comment:
Hi Chandana,

This was discussed in one of my code reviews and this was renamed as per Udara's comment.

Thanks,